src/pages/examSetter/ExamInformation.tsx

```
@@ -0,0 +1,187 @@
+import { ExamRequest } from '../../api/types';
+import { getLoggedInUser } from '../../utils/authUtils';
+import { useEffect, useState } from 'react';
+
+import { getExamInformation, saveExamInformation, updateExamInformation } from '../../api/services/ExamServices';
+import moment from 'moment';
+
+type ExamInformationProps = {
+    onFinishFun: (values: ExamRequest) => void;
+};
+
+export const ExamInformation: React.FC<ExamInformationProps> = ({ }) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)